modules/caffeCoder/src/main.cpp

```diff
@@ -91,7 +91,7 @@ class CaffeCoderPort: public BufferedPort<Image>
 
     CaffeFeatExtractor<float>    *caffe_extractor;
 
-    void onRead(ImageOf<PixelRgb> &img)
+    void onRead(Image &img)
```

Comment:
@jchevrie Actually this change was introduced in the recently merged PR by @Tobias-Fischer ([commit](c68eb29a5668e559418d19a891691932d44bb93f)) , I think we can ask him the reason why he introduced it before reverting it?

Reply:
I have not been using this code since a while and I would need some time to test it. Could you and @jchevrie come up with a version that compiles and works properly in both settings?